mlir/test/Dialect/Tosa/canonicalize.mlir

```diff
@@ -591,3 +591,13 @@ func.func @fold_abs_abs(%arg0: tensor<?x1xf32>) -> tensor<?x1xf32> {
   %1 = tosa.abs %0 : (tensor<?x1xf32>) -> tensor<?x1xf32>
   return %1 : tensor<?x1xf32>
 }
+
+// -----
+
+// CHECK-LABEL: @rank_zero
```

Comment:
Can you make a change to make the name more descriptive? Maybe something like fold_reduce_rank_zero or something similar.

Reply:
Sure, I renamed it to `fold_reduce_rank_zero`.